Source/Xamarin/Prism.Forms/Mvvm/ViewModelLocator.cs

```diff
@@ -35,8 +35,7 @@ public static void SetAutowireViewModel(BindableObject bindable, bool? value)
 
         private static void OnAutowireViewModelChanged(BindableObject bindable, object oldValue, object newValue)
         {
-            bool? bNewValue = (bool?)newValue;
-            if (bNewValue.HasValue && bNewValue.Value)
+            if (newValue is bool shouldAutowire && shouldAutowire)
```

Comment:
I might not be used yet to the C# 7 syntax, but I think the old syntax is easier to read/understand than this.